hb_config.sample.py

```diff
@@ -37,8 +37,11 @@
 max_user_age = 2
 
 # The minimum number of edits that a user must have made to qualify
-# for invitation.
-min_edits = page_reader.read_json_value(
+# for invitation. Note that the syntax for using the return value of a
```

Comment:
Add something here (ore elsewhere in the config) about it not being possible to use keyword arguments.